src/aap_eda/settings/default.py

```diff
@@ -636,6 +643,17 @@ def get_rulebook_process_log_level() -> RulebookProcessLogLevel:
 # Organization and object roles will come from create_initial_data
 ANSIBLE_BASE_ROLE_PRECREATE = {}
 
+# --------------------------------------------------------
+# GATEWAY RESOURCE API CLIENT
+# --------------------------------------------------------
+RESOURCE_SERVER = {
```

Comment:
@bzwei Since all our keys have an EDA prefix would the RESOURCE_SERVER__URL have to be called EDA_RESOURCE_SERVER__URL when passing in via env var. Should DAB parameters have their own prefix. Can dynaconf support multiple prefix EDA,DAB
https://github.com/ansible/eda-server/blob/01746bec37107c64bd0e11a58adb6600015e6951/src/aap_eda/settings/default.py#L65

Reply:
@bzwei @rochacbruno @Alex-Izquierdo Would it be better to call this **EDA_DAB_RESOURCE_SERVER__URL** to keep the DAB parameters separate.  Or **DAB_RESOURCE_SERVER__URL**. Since these are things specific to DAB. Or would the SyncExecutor() have issues locating these [parameters](https://github.com/ansible/django-ansible-base/blob/9f0f05168dbcb18c1f542aec480b80eedb5ffc3e/ansible_base/resource_registry/tasks/sync.py#L68) since it won't have the correct prefix.

Reply:
We already take DAB settings [here](https://github.com/ansible/eda-server/blob/main/src/aap_eda/settings/default.py#L603) So if we are overwriting the RESOURCE_SERVER from DAB, the double score makes no sense. The double score is for dynaconf to handle nested values. 
So in this case I see two options. 
1. We expose separated values, and RESOURCE_SERVER["url"] can be as it is now (but without double score)
```
RESOURCE_SERVER = {
    "URL": settings.get("RESOURCE_SERVER_URL", "https://localhost"),
    ....
}
```

We expose RESOURCE_SERVER as object, in that way the envvar `EDA_RESOURCE_SERVER__URL` would override only `RESOURCE_SERVER["url]`. 

```
default_resource_server = {
    "URL": "https://localhost",
    "SECRET_KEY": ""),
    "VALIDATE_HTTPS": False),
}
RESOURCE_SERVER = settings.get("RESOURCE_SERVER", default_resource_server)
```

Since it is not a complex object and we need to define all these values in the most of cases, I vote to keep them separately as it is now. 

All the settings exposed has automatically added the EDA prefix, adding the "DAB" prefix to allow us to know that it is a setting that is passed to DAB, seems a bit redundant to me, it is information for the development but has no meaning in the context of the application. For EDA, it's not the "DAB_RESOURCE_SERVER" it's just the resource server. 

To me it's fine as it is called now, in the settings file we have comments indicating that those values are for DAB.

Reply:
After more discussions and get more context, it seems the double underscore would be to be aligned with other components, so I guess in that case, the current approach would be fine as well.

Reply:
@Alex-Izquierdo I don't see the imported DAB settings include RESOURCE_SERVER. Can you point me the origin? If the RESOURCE_SERVER is already imported we should not overwrite.
> We already take DAB settings [here](https://github.com/ansible/eda-server/blob/main/src/aap_eda/settings/default.py#L603) So if we are overwriting the RESOURCE_SERVER from DAB, the double score makes no sense.